src/pages/Navgurukul/merakiAdmission/index.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { toast } from "react-toastify";
+import YouTube from "react-youtube";
+import { METHODS } from "../../../services/api";
+import "./styles.scss";
+
+function Admission() {
+  const [userDetails, setUserDetails] = useState({
+    firstName: "",
+    middleName: "",
+    lastName: "",
+    mobileNumber: "",
+  });
+
+  const [mobile, setMobile] = useState("");
+  const [enrolmentKey, setEnrolmentKey] = useState("");
+
+  const [partnerId, setPartnerId] = useState("");
+
+  let userToken = localStorage.getItem("Token");
+  useEffect(() => {
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}/users/me`,
+      headers: {
+        accept: "application/json",
+        Authorization: userToken,
+      },
+    }).then((res) => {
```

Comment:
This api will get call with the token which we are getting from android and if we won't get a token from android then only it will get call with token from user/me.

Reply:
Sorry, I wasn't being very clear with all my questions, API call?  What I meant is that as per the guidelines in the repository README file (https://github.com/navgurukul/bhanwari-devi#how-to-do-an-api-call-and-manage-state), I thought that all these `axios` API calls would ideally be made from a file `api.js` in a redux folder.  In the case here, the only difference between the call made in this file and that from index.js of Profile was the token that was used.  So we could add a function to api.js like this:

```
export const getProfileData = (token) => {
  return axios({ 
   method: METHODS.GET, 
   url: `${process.env.REACT_APP_MERAKI_URL}/users/me`, 
   headers: { 
     accept: "application/json", 
     Authorization: token, 
   }, 
 })
```

In this call, `getProfileData` is called with a `token` of `userToken` whereas on the Profile page, it's called with the value of `user.data.token`.  More abstraction can be done with redux/saga, but this could be a start.

Reply:
@jschanker actually we don't know Redux. Neither @komalahire knows nor I. If you can ever get the time to make us understand we'll do it.

Reply:
@Poonam-Singh-Bagh @komalahire Sure, happy to try to help you learn.  Here's a link to some interactive exercises that can help you get started: https://www.freecodecamp.org/learn/front-end-development-libraries/#redux .  And I can try to answer your questions as you go through the lessons and connect them to how you're using Redux here.

Reply:
Thank you @jschanker, where can we communicate? Are you on NavGurukul or Meraki slack?